examples/programmatic-highlighting/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Charts Embedded SDK Demo: Interactive Filtering</title>
+  <meta charset="UTF-8" />
+  <link rel="stylesheet" type="text/css"
+    href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" />
+  <link href="https://fonts.googleapis.com/css?family=Heebo&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" type="text/css" href="assets/styles.css" />
+</head>
+
+<body>
+  <main>
+    <h2>Change highlights for both charts</h2>
+    <div class="body-content">
+      <div class="chart" id="chart1"></div>
+      <div class="chart" id="chart2"></div>
+    </div>
+    <div class="options">
```

Comment:
Can we add a label along the lines of:
"Choose a predefined filter document to highlight the chart data, or enter your own MQL filter"

Reply:
Change made in bcaf65f